lib/plenario_web/controllers/web/page_controller.ex

```diff
@@ -162,15 +281,15 @@ defmodule PlenarioWeb.Web.PageController do
     ])
   end
 
-  @red "255, 99, 132"
-  @blue "54, 162, 235"
-  @yellow "255, 206, 86"
-  @green "75, 192, 192"
-  @purple "153, 102, 255"
+  @red "255,99,132"
+  @blue "54,162,235"
+  @yellow "255,206,86"
+  @green "75,192,192"
+  @purple "153,102,255"
```

Comment:
I'd like to put a pin in moving this at least to a view or template, but ideally directly into CSS or at least JS. Defining colors in the controller gives the middle finger to separation of concerns.

Reply:
The users select the color that they want for the graph. I had given this some thought and this is the compromise I came up with.

Reply:
I have some strong concerns over the utility and maintainability of giving users that option. Beyond the fact that it seems unnecessary to me, how do we handle comparing data sets/fields if they're both the same color? We'd have to override the user's color choice, and that's something people get irrationally upset by if they've been given an option in the first place.

There's an entire second layer to this that has to deal with stupid design decisions in Chart.js (i.e. all styling is done in JavaScript because no one contributing to the largest JS charting library sees anything wrong with that apparently), but that's pretty fully in my wheelhouse. I'm more concerned here with why we're giving this control to the users.

Reply:
So let's address this by component:

### Fill

Not every area of a line chart (the most popular option) should be filled, nor should they all be not filled. It's impossible to guess which it should be. Having the option to fill an area seems like a reasonable control to give the user.

### Color

In a similar vein to fill, users may want to select colors to highlight and coordinate datasets. Since ChartJS requires colors to be defined (otherwise they're all gray), and those colors are then handled in JS rather than CSS, it makes sense to just hand the control over the person defining the chart.

Reply:
Just want to throw in my two cents here, we can still allow users to define their chart colors and keep the logic for that entirely in the view layer. Keeping color information around in the controller doesn't sit well with me given that it's information that pretty much only affects aesthetics.

Reply:
I took another look at the context of this -- the code that you all are upset about is specifically there to build a cycle of colors to be applied to aggregates for AoT data. This has nothing to do with users picking colors.

I agree that it sucks here. But this PR is already massive enough.